govc/flags/library.go

```diff
@@ -91,3 +96,30 @@ func ContentLibraryItem(ctx context.Context, c *rest.Client, path string) (*libr
 	}
 	return &item, nil
 }
+
+// ContentLibrarySecurityPolicyFlag allows the user to specify the security policy ID when performing content library operations.
+type ContentLibrarySecurityPolicyFlag struct {
+	common
+
+	// The unique identifier of a security policy.
+	policy string
+}
+
+// NewSecurityPolicyFlag instantiates a command line flag for a content library security policy.
+func NewSecurityPolicyFlag(ctx context.Context) (*ContentLibrarySecurityPolicyFlag, context.Context) {
```

Comment:
I don't think this is needed.. the govc flags that need to use `Context.Value` are just those that may be embedded in flag structs more than once (Client, Output, Datacenter being the most common examples).
But in this case, the flag is only used by `library.policy.ls` and can just be added via `FlagSet.StringVar` via that command's `Register` function.

Reply:
Ack thanks. That's helpful background. I'll update to `StringVar`.